### ISSUE TITLE
'numpy.ndarray' object has no attribute 'dim'

### ISSUE THREAD

**Issue by kulasimu** [yongqyu/MolGAN-pytorch#8]
Hi, when I try to run the program with test mode, the error occurs. It shows 'numpy.ndarray' object has no attribute 'dim'. The error is located at solver.py, line 376, "edges_logits, nodes_logits = self.G(z)"
I tried several solutions on google but they didn't work. Could someone tell me what's the problem?

**Comment by yateeshagrawal** [comment]
In "def test(self):", z needs to be converted to torch tensor. So simply add "z = torch.from_numpy(z).to(self.device).float()" in a new line just after "z = self.sample_z(a.shape[0])".
